numba/np/ufunc/array_exprs.py

```diff
@@ -376,9 +376,10 @@ def _lower_array_expr(lowerer, expr):
             inner_sig_args.append(argty)
     inner_sig = outer_sig.return_type.dtype(*inner_sig_args)
 
+    flags = utils.ConfigStack().top_or_none() or compiler.Flags()
+    flags = flags.copy() # make sure it's a clone or a fresh instance
```

Comment:
```suggestion
    flags = utils.ConfigStack().top_or_none()
    flags = compiler.Flags() if flags is None else flags.copy() # make sure it's a clone or a fresh instance
```
does this save copying a fresh instance in the case TOS is None?